camayoc/tests/qpc/yupana/test_connect.py

```diff
@@ -0,0 +1,41 @@
+"""Starting tests for configs"""
+
+import requests
+
+from camayoc.tests.qpc.yupana.utils import (
+    yupana_configs,
+    get_app_url,
+    post_file,
+    get_x_rh_identity,
+)
+
+
+EXPECTED_CONFIGS = ("app_name", "openshift_project", "openshift_app_domain", "api_path")
+
+
+def test_config():
+    """test load configurations"""
+    config = yupana_configs()
+    assert config != []
+    assert sorted(set(config["yupana-app"].keys())) == sorted(EXPECTED_CONFIGS)
```

Comment:
Same thing, it is a good-thing (tm) to provide feedback on assertions errors, so we can see them when running the tests.